compose.yaml

```diff
@@ -15,3 +15,17 @@ services:
         SPRING_APPLICATION_JSON={"customerService":{"hostname":"customer-service","port":8081}}
     ports:
       - "8080:8080"
+
+  poss-db:
```

Comment:
This creates one DB, for all micro-services, which deviates from the specification:

> For Database, to continue with our isolation we chose decentralized data management, meaning that every microservice will have its own database. This makes not only the logic side of the microservice independent, but also the data side, making specific parts of our system completely independent and we can scale them at different paces.

Reply:
solved in https://github.com/domanteli0/PoSS/pull/4/commits/c47c539fb3e53bdf2b354778c92d17d1b1a99882